tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
-    "combat_state_poison_damage": "${{name}} estis venenigita!",
-    "combat_status_damage": "${{name}} estis damaĝita de ${{status}}!",
+    "combat_state_poison_damage": "${{name}} veneniĝis!",
+    "combat_status_damage": "${{name}} damaĝiĝis pro ${{status}}!",
     "combat_victory": "Vi gajnis!",
     "combat_defeat": "Vi estis venkita!",
-    "combat_capturing success": "Vi kaptis ${{name}}!",
+    "combat_capturing success": "${{name}} estias kaptita!",
```

Comment:
Until the name of Tuxemon are not localized we should use the passive forme to avoid accusative. I dislike it so much but we are obliged to do so in order to avoid words like "Spookthingn". Moreover, I hope that will never be Tuxemon with a space in the name: the accusative form would be "Spook thingn". OMG
By now I'd keep the passive form. We could discuss it later. ;)